### DISCUSSION TITLE
Keyboard blocking is irritating.

### DISCUSSION THREAD

**Discussion by ahiemstra** [isontheline/pro.webssh.net#958] [Ideas]
A lot of times I find the SSH session in copy/paste mode, which means I cannot type with my physical keyboard. I have to either tap the screen or tap the keyboard at the top in order to type text.

This is VERY irritating. 

The default should be typing and if I want to copy I click the keyboard.

If the session is in copy/paste mode, it should automatically go out of this mode after x seconds.

It should NEVER go into this mode without clicking the keyboard icon.

Regards, Arjen.

**Comment by isontheline** [comment]
Are you using the App Store release or Testflight one?

I've done some changes on beta : 
https://testflight.apple.com/join/QSrBK59z
